packages/xo-web/src/common/xo/migrate-vms-modal/index.js

```diff
@@ -121,16 +121,16 @@ export default class MigrateVmsModalBody extends BaseComponent {
     const { doNotMigrateVdi, doNotMigrateVmVdis, migrationNetworkId, networkId, smartVifMapping, srId } = this.state
 
     // Map VM --> ( Map VDI --> SR )
+    // 2021-02-16: Always build map VDI -> SR: to not migrate VDIs to the wrong SRs.
+    // Intra pool: the VDI will be migrated to the selected SR if it's on a local SR.
+    // Different pool: all VDIs will be migrated to the selected SR.
```

Comment:
```suggestion
    // 2021-02-16: Fill the map (VDI -> SR) with *all* the VDIs to avoid unexpectedly migrating them to the wrong SRs:
    // - Intra-pool: a VDI will only be migrated to the selected SR if the VDI was on a local SR.
    // - Inter-pool: all VDIs will be migrated to the selected SR.
```